images/samba/Dockerfile.fedora

```diff
@@ -0,0 +1,43 @@
+FROM quay.io/phlogistonjohn/sambacc:default AS builder
+
+# the changeset hash on the next line ensures we get a specifc
+# version of sambacc. When sambacc actually gets tagged, it should
+# be changed to use the tag.
+RUN /usr/local/bin/build.sh c8c419ecb406
```

Comment:
What is `/usr/local/bin/build.sh`?

Reply:
It's part of the sambacc project and should be included in the sambacc image. It's even that image's entrypoint.. See:  https://github.com/samba-in-kubernetes/sambacc/blob/master/tests/container/Containerfile   I can't see why it should be missing.

Locally:
```
podman run --entrypoint ls  quay.io/phlogistonjohn/sambacc:default   /usr/local/bin/build.sh
/usr/local/bin/build.sh
```
Is this a docker/podman compatibility issue? Does it work if you build it with podman instead?

Reply:
Hmm, my problem was with building the `Dockerfile.fedora`, not running a container. But I just created a build target to test this myself (because this PR doesn't have one), and maybe that was just wrong...

Reply:
Update: Works on my linux box with podman. Error was on a Mac with docker. Can be addressed later. 😄